data/decks/linne.json

```diff
@@ -0,0 +1,190 @@
+{
+	"id": "linne",
+	"season": 6,
+	"display_name": "Linne",
+	"cardback": "cardback_linne.jpg",
+	"exceed_cost": 1,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "draw",
+			"amount": 2,
+			"and": {
+				"effect_type": "draw",
+				"opponent": true,
+				"amount": 1,
+				"and": {
+					"effect_type": "choice",
+					"choice": [
+						{ "effect_type": "advance", "amount": 1 },
+						{ "effect_type": "retreat", "amount": 1 }
+					]
+				}
+			}
+		}
+	}],
+	"character_action_exceeded": [
+		{
+			"gauge_cost": 0,
+			"force_cost": 0,
+			"can_continuous_boost_from_gauge": true,
```

Comment:
maybe "requires_continuous_boost_in_gauge" would be more descriptive?

Reply:
I don't _think_ this distinction matters with the card pool we have, but if Linne or Wagner had boosts with force costs and no ability to pay them that would also prevent this action. Perhaps something like "can_play_continuous_boost_from_gauge"? Given I think the difference is academic, don't feel strongly if you prefer requires.

Reply:
yea, let's go with "can_play" then

Reply:
actually, what you have now is fine